src/lib/utils.ts

```diff
@@ -1,6 +1,21 @@
 import { clsx, type ClassValue } from "clsx"
 import { twMerge } from "tailwind-merge"
+import { client } from "@/lib/thirdwebClient";
+import { getCurrencyMetadata } from "thirdweb/extensions/erc20";
+import { getContract, defineChain} from "thirdweb";
+import { defaultChainId } from "@/lib/constants";
 
+const chain = defineChain(defaultChainId);
```

Comment:
can you define the full chain in constants, instead of just the chain id. call it `defaultChain`, export it and use it everywhere